### PR TITLE
Add further spanish translation

### PR DESCRIPTION
**Pull request by DavideDunne**
Add some translation for blog SEO and sidebar and theme navbar

@Nojipiz can you review the proposed changes?

### CODE REVIEW

File: i18n/es/docusaurus-theme-classic/navbar.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "title": {
+    "message": "LunarVim",
+    "description": "The title in the navbar"
+  },
+  "item.label.Docs": {
+    "message": "Documentacion",
```
Comment (Nojipiz) [inline i18n/es/docusaurus-theme-classic/navbar.json:7]: Is there a problem if we put accent marks here?
In this case it should be "Documentación"

**Review by Nojipiz — APPROVED** [review]
Gracias ;)

File: i18n/es/docusaurus-theme-classic/navbar.json (hunk 1 above)
Comment (DavideDunne) [reply i18n/es/docusaurus-theme-classic/navbar.json:7]: No problem with that, I thought that navbar and the font would have some issues with UTF-8, but I guess it's fine

Comment (Nojipiz) [reply i18n/es/docusaurus-theme-classic/navbar.json]: Let's hope it doesn't cause problems haha.
Nice PR

**Comment by rebuilt** [comment]
Gracias!

**Pull request merged**